### PR TITLE
🐧 [ubuntu] Remove some packages that should not be needed

### PR DESCRIPTION
**Pull request by sdwilsh**
**What this PR does / why we need it**:
In https://github.com/kairos-io/kairos/pull/821 a number of packages were explicitly added that were previously installed automatically.  This change removes some of the more obvious ones that we should not need to install manually.

### CODE REVIEW

**Comment by sdwilsh** [comment]
Here's the delta's on size of packages installed

| branch | ubuntu | ubuntu-22-lts | ubuntu-20-lts |
| --- | --- | --- | --- |
| master | 2061 MB | 1855 MB | 1739 MB |
| reduce-deps | 1898 MB (-7.9%) | 1699 MB (-8.4%) | 1521 MB (-2.3%) |

File: images/Dockerfile.ubuntu [hunk 1]
```diff
@@ -11,10 +11,8 @@ RUN apt-get update \
   && add-apt-repository ppa:oibaf/test \
   && apt-get update \
   && apt-get install -y --no-install-recommends \
-    binutils \
     conntrack \
     console-data \
-    console-setup \
```
Comment (mudler) [inline images/Dockerfile.ubuntu:17]: I think we need `console-setup`, IIRC for displaying correctly the QR code: #212

Comment (sdwilsh) [reply images/Dockerfile.ubuntu:17]: Based on the PR that fixed that issue, we need `console-data`, which this change keeps.

`console-setup` is not recommended by `console-data` either: https://packages.ubuntu.com/kinetic/console-data

Comment (sdwilsh) [reply images/Dockerfile.ubuntu:17]: I can't test this to verify it because I can't seem to build isos locally.  I've put more details in #875.

Comment (mauromorales) [reply images/Dockerfile.ubuntu:17]: Just built a new image without `console-setup` and it shows the QR properly.

**Review by mauromorales — APPROVED** [review]
lgtm!

**Pull request merged**